docs/nlu/training-data-format.rst

```diff
@@ -74,7 +77,8 @@ The most important one is ``common_examples``.
             "common_examples": [],
             "regex_features" : [],
             "lookup_tables"  : [],
-            "entity_synonyms": []
+            "entity_synonyms": [],
+            "intent_keywords": {}
```

Comment:
i think it's a bit random to have a separate section for this in the json file. Shouldn't it also just go under common examples like in the markdown?

Reply:
I think this is neater and probably simpler when the data is not managed manually, also the keywords are not examples and neither are they tied to examples the way entities are.
So I'd leave it like this but having a consistent standard is of course also important.

Reply:
This is also resolved by what we agreed to about the format